src/main/components/RequestBar.jsx

```diff
@@ -9,36 +10,50 @@ const RequestBar = (props) => {
   const [selected, setSelected] = useState(method);
   const [uri, setUri] = useState('');
 
+  // header info
+  const [headerType, setHeaderType] = useState('Authorization');
+  const [authType, setType] = useState('Bearer Token');
+  const [headerKey, setHeaderKey] = useState('');
 
   const handleChange = (e) => {
     const { name, value } = e.target;
     if (name === 'method') setSelected(value);
```

Comment:
As you mentioned, @Donaldblanc let’s plan to refactor this to a switch statement on a future iteration. Created an issue so we don’t forget (#40)